sentinel-transport/sentinel-transport-common/src/main/java/com/alibaba/csp/sentinel/transport/init/HeartbeatSenderInitFunc.java

```diff
@@ -72,13 +72,11 @@ private void setIntervalIfNotExists(long interval) {
     long retrieveInterval(/*@NonNull*/ HeartbeatSender sender) {
         Long intervalInConfig = TransportConfig.getHeartbeatIntervalMs();
         if (isValidHeartbeatInterval(intervalInConfig)) {
-            RecordLog.info("[HeartbeatSenderInitFunc] Using heartbeat interval "
-                + "in Sentinel config property: " + intervalInConfig);
+            RecordLog.info("[HeartbeatSenderInitFunc] Using heartbeat interval in Sentinel config property: {}" , intervalInConfig);
```

Comment:
please remove the unnecessary indent before ,

Reply:
done